packages/office-ui-fabric-react/src/components/Dialog/DialogFooter.base.tsx

```diff
@@ -1,13 +1,32 @@
 import * as React from 'react';
-import { BaseComponent, css } from '../../Utilities';
-import * as stylesImport from './Dialog.scss';
-const styles: any = stylesImport;
+import {
+  IDialogFooterProps,
+  IDialogFooterStyleProps,
+  IDialogFooterStyles,
+} from './DialogFooter.types';
+import {
+  BaseComponent,
+  classNamesFunction,
+  customizable,
+  IClassNames
+} from '../../Utilities';
 
+const getClassNames = classNamesFunction<IDialogFooterStyleProps, IDialogFooterStyles>();
+
+@customizable('DialogFooter', ['theme'])
 export class DialogFooterBase extends BaseComponent<any, any> {
+  private _classNames: IClassNames<IDialogFooterStyles>;
+
   public render() {
+    const { getStyles, theme } = this.props;
+
+    this._classNames = getClassNames(getStyles!, {
+      theme: theme!,
```

Comment:
I think if it is a Component class root, it should have a className style prop. With normal usage, you wont really pass anything to it, but DialogFooter is in the index.ts so a dev can grab use it in a custom way and if they do they should be able to pass use the className prop.